src/main/java/accounting/resources/ReportResource.java

```diff
@@ -0,0 +1,66 @@
+package accounting.resources;
+
+import io.dropwizard.db.DataSourceFactory;
+import net.sf.jasperreports.engine.*;
+
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.StreamingOutput;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+
+@Path("/")
+public class ReportResource {
+
+	private DataSourceFactory dsf;
+
+    @Inject
+    public ReportResource(DataSourceFactory dsf) {
+    	this.dsf = dsf;
+    }
+
+    @SuppressWarnings("deprecation")
+    @GET
+    @Path("/pdf-report")
+    @Produces({"application/pdf"})
+	public StreamingOutput generatePDF() throws JRException, IOException, SQLException {
+        String reportTemplate = "src/main/resources/ReportTemplate.jrxml";
+
+        Date month = new Date();
```

Comment:
This stuff we can replace with LocalDate so that we don't have to use deprecated stuff.  I made the changes locally to fix this stuff, so I can push that.  Basically just used LocalDate.now() and then I could use month.getMonth().getValue() (you won't need the + 1 in this case) and month.getYear() (you won't need the + 1900 in this case).